examples/streaming-examples.ts

```diff
@@ -0,0 +1,34 @@
+/* Initialize your client with your audio configuration and access token */
+const audioConfig = new AudioConfig(<YOUR-CONTENT-TYPE-PARAMS>);
+const token = <YOUR-ACCESS-TOKEN>;
+var client = new RevAiStreamingClient(token, audioConfig);
+
+/* Create your event responses */
+client.on('close', (code, reason) => {
+	console.log(`Connection closed, ${code}: ${reason}`);
+});
+client.on('httpResponse', code => {
+	console.log(code);
```

Comment:
Now that this is a public example and people will likely run it before they read it closely. Lets add a little more detail to these log statements